mlir/lib/Dialect/Arith/Utils/Utils.cpp

```diff
@@ -302,6 +302,13 @@ Value mlir::createScalarOrSplatConstant(OpBuilder &builder, Location loc,
   return builder.createOrFold<arith::ConstantOp>(loc, type, splat);
 }
 
+Value mlir::createIntOrIndexConstant(OpBuilder &b, Location loc, Type type,
+                                     int64_t value) {
+  assert(type.isIntOrIndex() &&
+         "unexpected type other than integers and index");
+  return b.create<arith::ConstantOp>(loc, b.getIntegerAttr(type, value));
+}
```

Comment:
Seems like this wrapper does not add any value: can we just inline this at use sites?

Reply:
Sounds good.